### PR TITLE
Show active token image rather than the prototype token

### PR DESCRIPTION
**Pull request by CarlosFdez**
Closes https://github.com/foundryvtt/pf2e/issues/16364

### CODE REVIEW

File: src/module/chat-message/document.ts [hunk 1]
```diff
@@ -199,11 +199,9 @@ class ChatMessagePF2e extends ChatMessage {
         if (actor && header && this.isContentVisible) {
             const token = this.token ?? actor.prototypeToken;
             const useToken =
-                actor.prototypeToken?.texture?.src &&
-                ImageHelper.hasImageExtension(token.texture.src) &&
-                !isDefaultTokenImage(token);
+                token?.texture?.src && ImageHelper.hasImageExtension(token.texture.src) && !isDefaultTokenImage(token);
```
Comment (stwlam) [inline src/module/chat-message/document.ts:202]: Is this nullable? The type may just need to be tighter if it's a `string`.

Comment (CarlosFdez) [reply src/module/chat-message/document.ts:202]: Its not, but it seems if you try feeding it a blank string foundry turns it into mystery man.

Comment (stwlam) [reply src/module/chat-message/document.ts:202]: _Can_ it be blank?

File: src/module/chat-message/document.ts [hunk 2]
```diff
@@ -199,11 +199,9 @@ class ChatMessagePF2e extends ChatMessage {
...
             const [imageUrl, scale] = useToken
-                ? [actor.prototypeToken.texture.src, Math.max(1, actor.prototypeToken.texture.scaleX)]
+                ? [token.texture.src, Math.max(1, token.texture.scaleX)]
                 : [actor.img, 1];
 
             const image = createHTMLElement("img");
```
Comment (stwlam) [inline src/module/chat-message/document.ts:207]:
```suggestion
            const image = document.createElement("img");
```

File: src/module/chat-message/document.ts (hunk 1 above)
Comment (CarlosFdez) [inline src/module/chat-message/document.ts:202]:
```suggestion
                token.texture.src && ImageHelper.hasImageExtension(token.texture.src) && !isDefaultTokenImage(token);
```

File: src/module/chat-message/document.ts [hunk 3]
```diff
@@ -199,14 +199,12 @@ class ChatMessagePF2e extends ChatMessage {
         if (actor && header && this.isContentVisible) {
             const token = this.token ?? actor.prototypeToken;
             const useToken =
-                actor.prototypeToken?.texture?.src &&
-                ImageHelper.hasImageExtension(token.texture.src) &&
-                !isDefaultTokenImage(token);
+                token.texture.src && ImageHelper.hasImageExtension(token.texture.src) && !isDefaultTokenImage(token);
```
Comment (stwlam) [inline src/module/chat-message/document.ts:202]:
```suggestion
                token.texture.src && !isDefaultTokenImage(token);
```

**Pull request merged**